pkg/virtctl/imageupload/imageupload.go

```diff
@@ -411,3 +421,17 @@ func getAndValidateUploadPVC(client kubernetes.Interface, namespace, name string
 
 	return pvc, nil
 }
+
+func getUploadProxyURL(client cdiClientset.Interface) (string, error) {
+	cdiConfig, err := client.CdiV1alpha1().CDIConfigs().Get(configName, metav1.GetOptions{})
```

Comment:
Do you need special handling for errors.IsNotFound()? Something like, "no default upload proxy configured, use --uploadproxy-url to pass it manually"? Is it supposed to always exist, or should the client gracefully work with older cdi versions (like e.g. a nice error message like I mentioned?

Reply:
It is supposed to always exist. CDIConfig is created when CDI is deployed since 1.7 release.
I can change the error message to a nicer message, please note that `virtctl` won't work with older CDI versions after this change.